src/lib/accessibility/Behaviors/Tab/TabBehavior.ts

```diff
@@ -0,0 +1,20 @@
+import { Accessibility } from '../../interfaces'
+
+const TabBehavior: Accessibility = (props: any) => ({
+  attributes: {
+    root: {
+      role: 'presentation',
+    },
+    anchor: {
+      role: 'tab',
+      'aria-selected': props['active'],
+      tabIndex: '0',
+      'aria-label': props['aria-label'],
+      'aria-labelledby': props['aria-labelledby'],
+      'aria-controls': props['aria-controls'],
+    },
+  },
+  handledProps: ['aria-label', 'aria-labelledby', 'aria-controls'],
```

Comment:
`aria-selected` should be introduced as `handled` one, corresponding changes to attributes logic should be provided as well:

```jsx
anchor: {
      ...
      'aria-selected': props['active'] || props['aria-selected'],
      ...
    },
```